packages/broker/src/broker.ts

```diff
@@ -33,7 +33,7 @@ export const createBroker = async (configWithoutDefaults: Config): Promise<Broke
             return streamrClient
         },
         start: async () => {
-            logger.info(`Start broker version ${CURRENT_VERSION}`)
+            logger.info(`Start Streamr node version ${CURRENT_VERSION}`)
```

Comment:
I guess `Streamr node` is the new name of broker? Wonder if we should still keep `broker` in parenthesis since the package name still has broker, i.e.
```suggestion
            logger.info(`Start Streamr (broker) node version ${CURRENT_VERSION}`)
```

Reply:
Yes please. We should drop 'broker' at every opportunity in favour of Streamr node.

Reply:
Sorry Eric, I didn't read this properly. 

Lets walk away from the Broker terminology. It should be `Start Streamr node version`
Hopefully one day even the package name :P